compat/server.js

```diff
@@ -9,7 +9,5 @@ try {
 	);
 }
 
-module.exports = {
-	renderToString: renderToString,
-	renderToStaticMarkup: renderToString
-};
+exports.renderToString = renderToString;
+exports.renderToStaticMarkup = renderToString;
```

Comment:
These are functionally identical, so I'm not quite sure why this was done?

Reply:
I've done it like that to keep consistency with client.js

Reply:
Consistency is a nice-to-have, but lets not change this here. It increases the diff size and messes with the blame unnecessarily.

Reply:
Okay changes done